test/integration/compare_sbom_input_vs_lib_test.go

```diff
@@ -48,9 +48,6 @@ func TestCompareSBOMInputToLibResults(t *testing.T) {
 
 	for _, image := range images {
 		t.Run(image, func(t *testing.T) {
-
-			t.Logf("Running case %s", image)
```

Comment:
Nice I did notice this was noisy when running some tests on main after seeing it changed in this PR.